scan-ui/src/main/res/drawable/bouncer_card_border_found.xml

```diff
@@ -1,16 +1,16 @@
 <animated-vector
-    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
not a problem, but thought I'd point it out. this is strange formatting. Most of this PR is just whitespace.

Reply:
whoa, that's weird. i'll check